README.md

```diff
@@ -53,7 +53,7 @@ After you have activated an API token, you have full access to the API.
 
 ## Example Usage
 
-To initialize an API session, issue the following commands:
+To initialize an API session, issue the following commands in Python:
```

Comment:
Perhaps a better solution is to indicate at the top of the file that all code should be run in `python`

Reply:
Some code before example usage are mix of git, bash and python.

Reply:
Fair point.